app/assets/stylesheets/main.scss

```diff
@@ -28,22 +29,70 @@ body {
   padding-right: 10px;
 }
 
+.btn.btn-primary {
+  color: white;
+}
+
+.btn.btn-danger {
+  color: white;
+}
+
 .glyphicon-star {
   color: #F7CA18;
 }
 
 // Custom CSS //
+
+div h1 {
+  color: #515253;
+  padding-bottom: 10px;
+}
+
+div .pull-right p {
+  margin-bottom: 15px;
+}
+
+.title {
+  padding-top: -10px;
+}
+
 .ice {
   height: 350px;
   margin-top: 5px;
-  padding: 10px;
-  box-shadow: 2px 1px 5px #ccc;
+  padding: 5px;
+  border-style: solid;
```

Comment:
Refactor `border-style`, `border-width`, and `border-color` to
 `border: 1px solid rgba(0,0,0,0.1);`